app/assets/javascripts/views/story_view.js

```diff
@@ -148,6 +148,12 @@ module.exports = FormView.extend({
         transitionEvent = key;
     })
 
+    if (transitionEvent === 'accept' || transitionEvent === 'reject') {
+      confirmed = confirm( I18n.t('story.definitive_sure', {action: transitionEvent} ));
```

Comment:
missing a `var` here, right? without it the `confirmed` will be set on `window`.